Jenkinsfile

```diff
@@ -317,9 +317,25 @@ void publishPublic() {
 }
 
 void publishMaven(mvnArgs='') {
-  withCredentials([usernamePassword(credentialsId: 'signing-creds', passwordVariable: 'SIGNING_PSW', usernameVariable: 'SIGNING_USR'),
-                   file(credentialsId: 'signing-key', variable: 'SIGNING_KEYFILE')]) {
-    sh "mvn deploy --settings build/jenkins.settings.xml -DskipTests ${mvnArgs}"
+  container('signing') {
+    withCredentials([certificate(credentialsId: 'cldtsdks-ciso-signing', keystoreVariable: 'CODE_SIGNING_PFX_FILE', passwordVariable: 'CODE_SIGNING_P12_PASSWORD')]) {
+      sh '''
+        #!/bin/bash -e
+        # Configure the client
+        setup-garasign-client
+
+        # Load GPG key from the server
+        GrsGPGLoader
+
+        # Place config in an expected location
+        cat << EOF > /home/jenkins/garasignconfig.txt
+name = GaraSign
+library = /usr/local/lib/Garantir/GRS/libgrsp11.so
+slotListIndex = 0
+EOF
```

Comment:
Can we indent these lines?

Reply:
we can, except `EOF`. done in https://github.com/IBM/cloudant-java-sdk/pull/413/commits/754f4b120d710881e49f41e86200be9056dbd748